server/actorController.js

```diff
@@ -39,21 +38,23 @@ const actorController = {
   },
 
   getActorCharacters: (req, res, next) => {
-    // TODO: add query property to change what table we are checking
-    const { firstName, lastName, option } = req.params;
+    const { actor } = req.query;
+    // skips over this middleware when there isn't a query
+    if (!actor) {
+      return next();
+    }
     let characterdb;
-    if (option === 'test') {
+    if (res.locals.title === 'test') {
```

Comment:
This is never set at this point since getCharacterData calls next() before setting it